select_func

```diff
@@ -0,0 +1,11 @@
+1. SELECT * FROM employees WHERE department_id = 50 AND salary > 4000;
+2. SELECT * FROM employees WHERE first_name LIKE 'A%';
+3. SELECT * FROM employees 
+WHERE department_id = 50 OR department_id = 80 
+AND commission_pct is not null;
+4. SELECT *, REPLACE(phone_number, '-', '.') FROM employees;
+5. SELECT department_id, salary, COUNT(*)
+FROM emplyees 
+GROUP BY department_id, salary
+HAVING COUNT (*) > 1;
+6. SELECT MAX(first_name) FROM employees;
```

Comment:
It's not correct!
We need to select employees with the longest first_name, not just MAX(first_name)

Reply:
Fixed. Make it with subquerry